DiscordBot/Commands/EventCommands.cs

```diff
@@ -19,34 +19,107 @@ internal class EventCommands
+
             await context.RespondAsync(
                 $"{context.Member.Mention} - choose one of the actions below or answer ``stop`` to cancel. (time out in 30s)\n" +
                 "``create`` - create new event.\n" +
                 "``rm`` - delete event.\n" +
                 "``edit`` - edit event.\n"
             );
 
-            var interactivity = context.Client.GetInteractivityModule();
-            var response = await interactivity.WaitForMessageAsync(
-                message =>
-                    IsValidResponse(message.Content),
-                TimeSpan.FromSeconds(30));
+            var eventOperation = await GetUserResponse(context, interactivity, EventOperations);
+            if (eventOperation == null)
+            {
+                return;
+            }
+
+            switch (eventOperation)
+            {
+                case "create":
+                    await CreateEvent(context, interactivity);
+                    break;
+                case "rm":
+                    await context.RespondAsync("Not implemented yet!");
+                    break;
+                case "edit":
+                    await context.RespondAsync("Not implemented yet!");
+                    break;
+            }
+        }
+
+        public async Task CreateEvent(CommandContext context, InteractivityModule interactivity)
+        {
+            await context.RespondAsync($"{context.Member.Mention} - what is the name of the event?");
+            var eventName = await GetUserResponse(context, interactivity);
+            if (eventName == null)
+            {
+                return;
+            }
+            
+            await context.RespondAsync($"{context.Member.Mention} - give an event description.");
+            var eventDescription = await GetUserResponse(context, interactivity);
+            if (eventDescription == null)
+            {
+                return;
+            }
 
-            if (response == null)
+            await context.RespondAsync($"{context.Member.Mention} - what time is your event?");
+            var eventTimeResponse = await GetUserResponse(context, interactivity);
+            if (eventTimeResponse == null)
             {
-                await context.RespondAsync($"{context.Member.Mention} - timed out");
+                return;
             }
-            else
+
+            if (!DateTimeOffset.TryParse(eventTimeResponse, out var eventTime))
+            {
+                await context.RespondAsync($"{context.Member.Mention} - operation stopped. (invalid date format)");
+                return;
+            }
+
+            var embed = new DiscordEmbedBuilder
+            {
+                Title = eventName,
+                Description = eventDescription,
+                Color = new DiscordColor(0xFFFFFF),
+                Timestamp = eventTime
+            };
+
+            await context.RespondAsync(embed: embed);
```

Comment:
What does the response look like? 🙂

Reply:
![image](https://user-images.githubusercontent.com/36454654/98833204-a977cf00-2435-11eb-8887-d6456e2fe45c.png)
For now the bot only responds with an embedded message with the details provided.

Reply:
Nice, that's a good start! 🙂